pkgs/development/interpreters/php/default.nix

```diff
@@ -204,8 +101,6 @@ let
           --replace '@PHP_LDFLAGS@' ""
       done
 
-      substituteInPlace ./build/libtool.m4 --replace /usr/bin/file ${file}/bin/file
```

Comment:
This replace don't need?
Error - `./configure: line 84378: /usr/bin/file: No such file or directory`

Reply:
Do you find anything that is broken due to removing this line other than this warning?

Having this line in adds `5.7M` to the closure size, and if that only is to get rid of a warning but actually not solve any issues, then we're not pulling file in.

Because 5.7M is 6% added closure size.

Reply:
ok